pytorch/tests/tests.yaml

```diff
@@ -30,6 +30,9 @@ import-cpu-oneccl-${PACKAGE_OPTION:-pip}:
 import-cpu-inc-${PACKAGE_OPTION:-pip}:
   img: ${REGISTRY}/${REPO}:b-${GITHUB_RUN_NUMBER:-0}-${BASE_IMAGE_NAME:-ubuntu}-${BASE_IMAGE_TAG:-22.04}-${PACKAGE_OPTION:-pip}-py${PYTHON_VERSION:-3.10}-ipex-${IPEX_VERSION:-2.3.0}-oneccl-inc-${INC_VERSION:-2.6}
   cmd: python -c "import neural_compressor as inc;print(inc.__version__)"
+import-cpu-transformers-${PACKAGE_OPTION:-pip}:
+  img: ${REGISTRY}/${REPO}:b-${GITHUB_RUN_NUMBER:-0}-${BASE_IMAGE_NAME:-ubuntu}-${BASE_IMAGE_TAG:-22.04}-${PACKAGE_OPTION:-pip}-py${PYTHON_VERSION:-3.10}-ipex-${IPEX_VERSION:-2.3.0}-hf-${HF_VERSION:-4.41.2}
+  cmd: python -c "import torch;import intel_extension_for_pytorch as ipex;import transformers;print(f'torch {torch.__version__} ipex {ipex.__version__} transformers {transformers.__version__}')"
```

Comment:
Do we need to print the torch/ipex info? We assume that it will work in the base layers.

Reply:
So just print transformers version then?

Reply:
Yes, this is more of a smoke test for just the packages introduced in this stage.